test/suites/storage.sh

```diff
@@ -57,7 +57,7 @@ test_storage() {
       lxc stop --force uuid1
       lxc stop --force uuid2
       if [ "$lxd_backend" = "lvm" ]; then
-        [ "$(blkid -s UUID -o value -p /dev/"${POOL}"/containers_uuid1)" != "$(blkid -s UUID -o value -p /dev/"${POOL}"/containers_uuid2)" ]
+        [ "$(blkid -s UUID -o value -p /dev/"${POOL}"/container_uuid1)" != "$(blkid -s UUID -o value -p /dev/"${POOL}"/container_uuid2)" ]
```

Comment:
That commit should be squashed with the other.

This also highlights an issue right there. I was expecting this change to only affect the API, making those links valid.
Instead it looks like this also affects the name of on-disk artifacts (in this case a LV name) without any logic to cope with the upgrade.

So when a user upgrades to this branch, LXD will be unable to find any of their exists containers as it will look for container_NAME rather than the existing containers_NAME.

Reply:
Perhaps we can keep support the old `containers` string, and use the new `container` string for new containers.

Reply:
That sounds like a bad idea as it'll force us to always look at two different places or we'll need to rename everything which means shutting down and restarting all containers on update.

Seems easier to me to leave the filesystem alone and only fix the API issue.